.github/workflows/tag_release.yaml

```diff
@@ -33,3 +36,5 @@ jobs:
         run: |
           git tag ${{ steps.tag.outputs.tag }}
           git push --tags
+      - name: Create release
+        run: gh release create {{ steps.tag.outputs.tag }} --verify-tag --generate-notes
```

Comment:
I don't really understand this change. The workflow should just create the tag and let the release workflow (with goreleaser) take care of creating the release with proper assets.